microservices/api-gateway/vapt/values.yaml

```
@@ -0,0 +1,71 @@
+name: "interop-be-api-gateway"
+techStack: "nodejs"
+
+service:
+  create: true
+  albHealthCheck:
+    path: "/api-gateway/0.0/status"
```

```suggestion
    path: "/api-gateway/1.0/status"
```
we must use the PROD values, this environment's configuration should mirror PROD as much as possible (except account-specific configuration of course)